src/renderer/components/layout/top-bar/top-bar.tsx

```diff
@@ -48,34 +54,10 @@ ipcRendererOn("history:can-go-forward", (event, state: boolean) => {
   nextEnabled.set(state);
 });
 
-export const TopBar = observer(({ children, ...rest }: Props) => {
+const NonInjectedTopBar: React.FC<Props & Dependencies> = (({ items, children, ...rest }) => {
```

Comment:
Let's stick with single way of setting `Props` notation as of
```
const NonInjectedTopBar = (({ items, children, ...rest }: Props & Dependencies) => 
```
Unless we have some benefits of using `React.FC`.

Reply:
Sure, done.